README.md

```diff
@@ -19,13 +19,18 @@ performance and reduce the memory overhead of its predecessors.  It is written
 entirely in C++ and is released under an open-source software license (see file
 [COPYING](https://github.com/CVC4/CVC4/blob/master/COPYING)).
 
+If you are using CVC4 in your work, or incorporating it into software of your
+own, we would like to invite you to send us a description and link to your
+project/software,
+so that we could put it on our [Third Party
+Applications](https://cvc4.github.io/third-party-applications.html) page.
 
 Website
 -------------------------------------------------------------------------------
-
-More information about CVC4 is available at:
+Documentation for users of CVC4 is available at:
 http://cvc4.cs.stanford.edu/
```

Comment:
Leave this as is, but introduce an extra section `Documentation`.
Then have one sentence and link for users to the website (ideally to the documentation section of the website), and then one sentence and link for developers to the GitHub Wiki.

Reply:
Changed that sentence a bit just to prevent duplication.
Currently the documentation page in the website is not merged, so I linked to the main page.
And for developers, did you mean the main wiki page or developer's guide? So far I chose the latter.

Reply:
LG. Which brings up the question, actually, if we need the front page of the Wiki -- we should move this information to the website. What do you think?